LEAF_Nexus/sources/Group.php

```diff
@@ -775,6 +802,9 @@ public function getPrivileges($groupID)
      */
     public function togglePermission($groupID, $categoryID, $UID, $permissionType)
     {
+        if(!is_numeric($groupID)||!is_numeric($UID)){
```

Comment:
There shouldn't be a check against `$UID` being numeric, since it's not always guaranteed that it will be numeric, it depends on the class.